packages/editor/src/components/inserter/index.js

```diff
@@ -94,26 +94,30 @@ export default compose( [
 			getBlockInsertionPoint,
 			getSelectedBlock,
 			getInserterItems,
-			getBlockOrder,
 		} = select( 'core/editor' );
-		const insertionPoint = getBlockInsertionPoint();
-		const parentId = rootClientId || insertionPoint.rootClientId;
+
+		let index;
```

Comment:
I think this should be intiailized something like getBlockOrder( rootClientId ).length?

Reply:
As far as I can tell, it's not needed, for reasons described in the comment a few lines below (maybe more valuable to pull up?)

>// Otherwise, the default behavior for an undefined index is to
>// append block to the end of the rootClientId context.

Reply:
Related:

https://github.com/WordPress/gutenberg/blob/a54896121a115ad143fd9b6ac331ca8eedf212af/packages/editor/src/store/reducer.js#L422-L428